setup.py

```diff
@@ -1,6 +1,6 @@
 from setuptools import setup, find_packages
 
-__version__ = "0.0.1"
+__version__ = "0.0.5"
 
 with open("hyperseti/version.py", "w") as fh:
```

Comment:
This is nice -- are there any circumstances where it could find the wrong file path?